experimental/import-source-phase.md

```diff
@@ -0,0 +1,29 @@
+# [Source Phase Imports][proposal-import-reflection]
+
+## Imports
+
+### ImportDeclaration
+
+```js
+extend interface ImportDeclaration {
+    phase: "source" | null;
+}
+```
+
+`phase` is `"source"` when representing an import in the form `import source X from "X"`.
+
+When `phase` is `"source"`, the `specifiers` must be a length-1 array including `ImportDefaultSpecifier`.
```

Comment:
Should we also add that "if `attributes` is defined, it must be an empty array"? Since `import source X from "foo" with { type: "json" }` is not a thing, right?